internal/synclet/synclet.go

```diff
@@ -19,8 +23,70 @@ func NewSynclet(dcli build.DockerClient) *Synclet {
 	return &Synclet{dcli: dcli}
 }
 
-func (s Synclet) GetContainerIdForPod(podId string) (string, error) {
-	return "", errors.New("GetContainerIdForPod not implemented")
+const pauseCmd = "/pause"
+
+func (s Synclet) GetContainerIdForPod(ctx context.Context, podId string) (string, error) {
```

Comment:
should be using k8s.PodID wherever possible -- it IS just a string, but (according to nick) a good way to make args transparent and enforce that we're passing not only a string but a VALID string...?